consensus/src/persistent_liveness_storage.rs

```diff
@@ -336,15 +358,44 @@ impl PersistentLivenessStorage for StorageWriteProxy {
             .get_data()
             .expect("unable to recover consensus data");
 
-        let last_vote = raw_data.0.map(|vote_data| {
-            bcs::from_bytes(&vote_data[..]).expect("unable to deserialize last vote msg")
+        let last_vote = raw_data.0.map(|bytes| {
+            // backward compatible for the 2-chain struct change
```

Comment:
nit: Do we really want to inline this here? Why not a backward_compatability.rs file?

Reply:
I agree, let's add a comment here or a reference to an issue once this migration is needed to be cleaned up with the new votes